packages/babel-plugin-transform-runtime/scripts/build-dist.js

```diff
@@ -67,6 +68,34 @@ function writeHelperFiles(runtimeName, { esm, corejs }) {
   }
 }
 
+function writeAutoHelperFiles(runtimeName) {
+  const pkgDirname = getRuntimeRoot(runtimeName);
+
+  for (const helperName of helpers.list) {
+    const filename = path.join(
+      pkgDirname,
+      "helpers",
+      "auto",
+      helperName,
+      "package.json"
+    );
+
+    outputFile(
+      filename,
+      JSON.stringify(
+        {
+          name: `${runtimeName}/helpers/auto/${helperName}`,
```

Comment:
Are the `name` and `private` fields needed? I'd have expected only `main` and `module`.

Reply:
They are probably not, Im just accustomed to using them. IMHO `private` gives an extra pinch of confidence that this won't be tried to get published (especially in lerna setup in case of some weird bug on their side).

`name` gives some more meaning to those files (kinda same with private, semantically both are ok). So I find adding those fields a nice touch, but as mentioned - they are not probably strictly required, so I can check it out and remove them if you feel they are not needed here.

Reply:
My feeling is that if `name` isn't there, then an accidental publish isn't a concern either way. If we really want `name`, then I agree having the `private: true` is a good choice though. I'd probably rather leave them out and add them if we get reports of it being an issue.

Reply:
Removed them.